packages/shared/src/features/onboarding/shared/FunnelStepper.tsx

```diff
@@ -77,15 +77,6 @@ function FunnelStepComponent<Step extends FunnelStep>(props: Step) {
   return <Component {...props} />;
 }
 
-function getTransitionDestination(
-  step: FunnelStep,
-  transitionType: FunnelStepTransitionType,
-): FunnelStepTransition['destination'] | undefined {
-  return step.transitions.find((transition) => {
-    return transition.on === transitionType;
-  })?.destination;
-}
```

Comment:
Found it to be simpler making it in-line. Can be found at the changes on line 114.